### ISSUE TITLE
Allow capturing job names by regex

### ISSUE THREAD

**Issue by lkuchlan** [nathan-weinberg/jeeves#94]
"blockers.yaml" file has "owners" field for each job, since we have multiple jobs, I think  it'll be more convenient to use regex.

For example:
- '^DFG-all-unified-{product_version}.*netapp-nfs-extra':
       owners:
          - owner1@website.com
          - owner2@website.com

If there is bz or jira for a specific job, we will need to specify separately, like:

- 'DFG-all-unified-16_director-rhel-virthost-3cont_2comp-ipv4-vxlan-netapp-nfs-extra':
        bz:
            - 123
        jira:
            - 'RHOSINFRA-123'

**Comment by odyssey4me** [comment]
Agreed, this would be great. Currently providing the same dict to multiple jobs is pretty painful.
